devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy

```diff
@@ -145,10 +145,10 @@ def ping() {
 
 def configure() {
 	log.debug "configure"
-	sendEvent(name: "checkInterval", value:6 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
 	Integer minReportTime = 0
 	Integer maxReportTime = 180
 	Integer reportableChange = null
+	sendEvent(name: "checkInterval", value: maxReportTime * 2 + 10 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```

Comment:
Is there a reason for `10 * 60` instead of `2 * 60`?

Reply:
@kianooshST thinks 10m is the better "fuzzing" number.

Reply:
That's for large checkin intervals